CHANGELOG.md

```diff
@@ -1,3 +1,155 @@
+# v1.13.0 (2022-03-15)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to Bottlerocket ([#2829])
+* rpm2img: stripe root filesystem for measureable performance improvement during boot ([#2863]) ([#2879])
+* Label `BOTTLEROCKET-DATA` at runtime ([#2807])
```

Comment:
```suggestion
* Create and set-up data partition at runtime ([#2807], [#2879], [#2873])
```

Reply:
Nice - thanks for the clarity on this one: I wasn't sure the best one-line explanation for it.